CarpetX/src/prolongate_3d_rf2_impl.hxx

```diff
@@ -187,33 +187,39 @@ template <typename T> struct coeffs1d<CC, POLY, /*order*/ 11, T> {
 
 // Hermite interpolation (with matched first derivatives)
 
-// Linear Hermite interpolation is the same as linear Lagrange interpolation
-template <typename T> struct coeffs1d<VC, HERMITE, /*order*/ 1, T> {
-  static constexpr std::array<T, 4> coeffs = {
-      +1 / T(2),
-      +1 / T(2),
-  };
-};
-// Cubic Hermite interpolation is the same as cubic Lagrange interpolation
 template <typename T> struct coeffs1d<VC, HERMITE, /*order*/ 3, T> {
-  static constexpr std::array<T, 4> coeffs = {
-      -1 / T(16),
-      +9 / T(16),
-      +9 / T(16),
-      -1 / T(16),
+  static constexpr std::array<T, 6> coeffs = {
+      +1 / T(96),
+      -9 / T(96),
+      56 / T(96),
+      56 / T(96),
+      -9 / T(96),
+      +1 / T(96)
   };
 };
 template <typename T> struct coeffs1d<VC, HERMITE, /*order*/ 5, T> {
-  static constexpr std::array<T, 6> coeffs = {
-      +121 / T(8192),  -875 / T(8192), +2425 / T(4096),
-      +2425 / T(4096), -875 / T(8192), +121 / T(8192),
+  static constexpr std::array<T, 8> coeffs = {
+      -28 / T(11520),
+      275 / T(11520),
+      -1377 / T(11520),
+      6890 / T(11520),
+      6890 / T(11520),
+      -1377 / T(11520),
+      275 / T(11520),
+      -28 / T(11520)
   };
 };
 template <typename T> struct coeffs1d<VC, HERMITE, /*order*/ 7, T> {
-  static constexpr std::array<T, 6> coeffs = {
-      -129 / T(32768),     +1127 / T(36864),    -6419 / T(49152),
-      +178115 / T(294912), +178115 / T(294912), -6419 / T(49152),
-      +1127 / T(36864),    -129 / T(32768),
```

Comment:
hmm, this for sure should not even have compiled, I count 8 elements in the initializer for a std::array of size 6.